instrumentation/spring/spring-rmi/javaagent/build.gradle.kts

```
@@ -0,0 +1,23 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.springframework")
+    module.set("spring-context")
+    versions.set("[5.0.4.RELEASE,5.4)")
```

since there's no 5.4 yet, it's good to leave unbounded, that way CI will fail when 5.4 is released if our instrumentation/tests fail on it

also, can you briefly describe why it only works starting at 5.0.4?

```suggestion
    versions.set("[5.0.4.RELEASE,)")
```

---

Actually nothing significant, the application on which I was testing the instrumentation is using `spring-starter-web`, and I tested till it's version 2.0.0.RELEASE, which comes bundled with `spring-context` version 5.0.4.RELEASE.
Now I tested it with earlier versions and the instrumentation works with `spring-context` version 4.0.0.RELEASE.
I will put out a new commit with the minimum version changed to 4.